apps/system/lockscreen/js/lockscreen_state_manager.js

```diff
@@ -415,10 +416,52 @@
         this.onUnlockingApp();
         break;
       case 'lockscreenslide-activate-right':
```

Comment:
we're going to be removing the slide for red square, so this event won't ever happen anymore, right?  Probably better to remove it than to add code for it!